.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: dart
 sudo: required
+addons:
+  chrome: stable
 
 dart:
   - 1.24.3
-  - stable
+  - dev
 dart_task:
+  - dartfmt: true
+  - dartanalyzer: lib example tool test/unit
   - test: --platform vm
-  - test: --platform dartium
-  - install_dartium: true
+  - test: --platform chrome
 
 script:
-  - pub publish --dry-run # Validate package
-  - pub run dart_dev gen-test-runner --check
-  - pub run dart_dev format --check
-  - pub run dart_dev analyze
+  #- xvfb-run --server-args="-screen 0 1024x768x24" tool/test.sh
   #- pub run dart_dev coverage --no-html
   - bash <(curl -s https://codecov.io/bash) -f coverage/coverage.lcov
```

Comment:
@robbecker-wf did you ever figure out if using `dart_task` results in the `script` section being skipped?

Reply:
It does not.

Reply:
I just discovered that, surprisingly, having a script section and a dart_task section results in only running the script section. So, in this file, you're not actually running tests, format, or analyzer. 😳

Reply:
:tableflip: